### ISSUE TITLE
Run the stdlib tests in presubmits

### ISSUE THREAD

**Issue by Verdagon** [ValeLang/Vale#466]

**Comment by Verdagon** [comment]
And make sure they dont supply a stdlib= mapping, so we can indirectly test that the stdlib is automatically brought in.

**Comment by tomerghelber** [comment]
Hey, this is a pretty important test.
If we want to split development on stdlib I think we should push this forward.

**Comment by Verdagon** [comment]
I agree! It's on my shortlist https://github.com/ValeLang/Vale/issues?q=assignee%3AVerdagon+is%3Aopen so it'll be done soon, likely within 2 weeks, so you caught this at a good time.
